<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaceInfoDialog</class>
 <widget class="QDialog" name="PlaceInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基站标定</string>
  </property>
  <widget class="QToolButton" name="btnComOpen">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>20</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="btnRead">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>读取</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnWrite">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>写入</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnOnekey">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>一键标定</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_role">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ANCHOR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TAG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CONSOLE</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>角色</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>331</width>
     <height>331</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>A0</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>A1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>A2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>A3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>A4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>A5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>A6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>A7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>A8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>A9</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>X/m</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Y/m</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Z/m</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ID">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>121</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
